app/views/test/index.html.erb

```diff
@@ -1,2 +1,212 @@
-<p>This is a test page. It will serve as the home page for logged in users.</p>
-<p>In the meantime, however, have a Weibo alien. <i class="fa fa-weibo"></i></p>
+<div class="col-md-8" style="padding:0px;">
+  <div class="panel panel-default" style="border-radius:15px;margin-left:10px;margin-right:10px;">
+    <div class="panel-heading" style="border-top-left-radius:15px;border-top-right-radius:15px;"><b>Videos In Progress</b> <a href="#" style="float:right;">+</a></div>
```

Comment:
This line is unnecessarily huge and unreadable. You can split it up.

Also, hardcoding CSS like this is not pretty. Instead of `style="float:right;"`, use `class="pull-right"`, which is a Bootstrap-defined CSS selector.